lib/checker_jobs/notifiers/logger.rb

```diff
@@ -0,0 +1,53 @@
+require 'logger'
+
+class CheckerJobs::Notifiers::Logger
+  DEFAULT = {
```

Comment:
Do we need a hash here to be consistent with other notifiers? Only if we don't, WDYT of using `DEFAULT_LEVEL`, ... ?

Reply:
Not sure we need to be consistent with other notifiers yet, we'll see how it goes I guess ;)
I'll change from a hash to 2 constants 👍